addons/minio/template/testgrid/k8s-docker.yaml

```diff
@@ -166,3 +166,84 @@
     minio_object_store_info
     validate_testfile rwtest testfile.txt
     validate_read_write_object_store postupgrade upgradefile.txt
+    dump_longhorn_logs
+
+# upgrade from 2020-01-25T02-50-51Z using openebs
+- name: upgrade minio from 2020-01-25T02-50-51Z (openebs)
+  flags: "yes"
+  installerSpec:
+    kubernetes:
+      version: "latest"
+    weave:
+      version: "latest"
+    openebs:
+      version: "latest"
+      isLocalPVEnabled: true
+      localPVStorageClassName: "default"
+    containerd:
+      version: "latest"
+    minio:
+      version: "2020-01-25T02-50-51Z"
+  upgradeSpec:
+    kubernetes:
+      version: "latest"
+    weave:
+      version: "latest"
+    openebs:
+      version: "latest"
+      isLocalPVEnabled: true
+      localPVStorageClassName: "default"
+    containerd:
+      version: "latest"
+    minio:
+      version: "__testver__"
+      s3Override: "__testdist__"
+  postInstallScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    minio_object_store_info
+    validate_read_write_object_store rwtest testfile.txt
+  postUpgradeScript: |
+    source /opt/kurl-testgrid/testhelpers.sh
+    minio_object_store_info
+    validate_testfile rwtest testfile.txt
+    validate_read_write_object_store postupgrade upgradefile.txt
+
+- name: upgrade minio from 2020-01-25T02-50-51Z while increasing PVC claim size (openebs)
+  flags: "yes"
+  installerSpec:
+    kubernetes:
+      version: "latest"
```

Comment:
Should we not install 1.23.x or 1.24.x here to do the check?

Reply:
I don't think this is very relevant because what we are testing here is the Minio upgrade.